bind-operator/src/bind.py

```diff
@@ -213,12 +216,13 @@ def update_zonefiles_and_reload(
                 pathlib.Path(constants.DNS_CONFIG_DIR) / f"db.{constants.ZONE_SERVICE_NAME}",
                 templates.ZONE_SERVICE.format(
                     serial=int(time.time() / 60),
+                    mailbox=config["mailbox"],
```

Comment:
Minor: Shouldn't it be validated? Any string is valid?

Reply:
The only restrictions I see are no `@` symbol and `.` should be escaped with a backslash. So it is very broad (it is an email after all, which is [difficult to properly validate](https://stackoverflow.com/questions/201323/how-can-i-validate-an-email-address-using-a-regular-expression) without using a long and obscure regex.  

Do you have an idea on how to do that ?

Reply:
I found one here https://github.com/canonical/landscape-server/blob/27263192a49088fa6a032cb234297d3288a3dcbf/canonical/lib/mail.py#L18
But I think validating for not empty and not special chars would be enough.

Reply:
I decided to go for some simple validation that results in stronger constraints than the RFC.